modules/cloudtrail-s3-to-cloudwatch/variables.tf

```diff
@@ -26,3 +26,27 @@ variable "cloudtrail_region" {
   default     = ""
   description = "The region CloudTrail reside. Used to to setup permissions on Bucket to accept logs from. Defaults to current region if non provided."
 }
+
+variable "kms_key_id" {
+  type        = string
+  default     = ""
+  description = "(Optional) The ARN of the KMS Key to use when encrypting log data. Please note, after the AWS KMS CMK is disassociated from the log group, AWS CloudWatch Logs stops encrypting newly ingested data for the log group. All previously ingested data remains encrypted, and AWS CloudWatch Logs requires permissions for the CMK whenever the encrypted data is requested."
+}
+
+variable "sse_algorithm" {
```

Comment:
same here have group of variables enabled, logging and sse_algorithm with optional() constructions